eslint.config.mjs

```diff
@@ -15,5 +16,10 @@ export default ts.config(
     languageOptions: {
       globals: globals.node,
     },
+    plugins: { "chai-friendly": pluginChaiFriendly },
+    rules: {
+      "@typescript-eslint/no-unused-expressions": "off", // disable original rule
+      "chai-friendly/no-unused-expressions": "error",
+    },
```

Comment:
use `pluginChaiFriendly.configs.recommendedFlat` instead of this

Reply:
```
"@typescript-eslint/no-unused-expressions": "off"
```

must stay as this is a workaround. `pluginChaiFriendly.configs.recommendedFlat` disables `no-unused-expressions` not `@typescript-eslint/no-unused-expressions`